src/index.js

```diff
@@ -176,37 +182,49 @@ function addMetadata({ types: t }) {
       CallExpression(babelPath, state) {
         if (!state.opts.shouldLoadFile) return;
 
-        // If we're at a top-level call expression, then we reset the beforeEachModified state to false, and let Babel
-        // move on to the next call expression.
-        if (babelPath.scope.block.type === 'Program') {
-          state.opts.beforeEachModified = false;
-          return;
+        // If this call expression is a top-level module, store string arg, else skip the nested module entirely
```

Comment:
I think you mean 'store module name string' rather than 'store string arg'.

```suggestion
        // If this call expression is a top-level module, store module name string, else skip the nested module entirely
```